### ISSUE TITLE
Data sources for promt-injection attacks

### ISSUE THREAD

**Issue by kashif2000** [ceterum1/llm-defender-subnet#27]
Can you provide list of data sources that narrate examples of all types of prompt injection attacks?

**Comment by ceterum1** [comment]
Absolutely, we'll be updating the documentation as we renew the prompt generation method.
